### ISSUE TITLE
Enable strict nullability checks

### ISSUE THREAD

**Issue by deffrian** [NethermindEth/nethermind#5537]
Right now we poorly handle nullable values because in some parts nullability checks is disabled.

**Comment by kamilchodola** [comment]
@rubo adjusted it recently for cli and env, right? 

@deffrian Is it about configuration settings or sth else?

**Comment by rubo** [comment]
@kamilchodola No, this is about nullable annotations in C#, and the issue is still actual.

**Comment by LukaszRozmej** [comment]
I think only few test projects left?
